app/src/main/java/com/example/bookclub/WebViewActivity.java

```diff
@@ -0,0 +1,72 @@
+package com.example.bookclub;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.graphics.Bitmap;
+import android.os.Bundle;
+import android.view.KeyEvent;
+import android.view.View;
+import android.webkit.WebSettings;
+import android.webkit.WebView;
+import android.webkit.WebViewClient;
+import android.widget.ProgressBar;
+
+public class WebViewActivity extends AppCompatActivity {
+    private WebView mWebView;
+    private Bundle bundle;
+    private ProgressBar mProgressBar;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_web_view);
+
+        mProgressBar = findViewById(R.id.progress_bar);
+        setupWebView();
+
+        bundle = getIntent().getExtras();
+        String idUrl = bundle.getString("id");
+        mWebView.loadUrl("openlibrary.org/isbn/" + idUrl);
```

Comment:
would make the url more specific.

~~~
        mWebView.loadUrl("https://openlibrary.org/isbn/" + idUrl);
~~~